### PR TITLE
Add 1.3.45 release notes

### PR DESCRIPTION
**Pull request by volllly**

### CODE REVIEW

**Review by Bluette-C-Riviere — APPROVED** [review]

File: middleware/2023-03-15-1.3.45.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+<!--truncate-->
+
+:::caution
+
+Version 1.3 of the Middleware is meant for the German market only, customers in Austria and France should continue to use version 1.2. We will unify these experiences in an upcoming version.
+
+:::
+
+## Feature: Respect HelipadHelper configuration in ByoDC
+We've integrated the HelipadHelper into ByoDC. This provides better upload stablilty and ByoDC now respects the HelipadHelper configuration of the cashbox.
```
Comment (TSchmiedlechner) [inline middleware/2023-03-15-1.3.45.md:18]:
```suggestion
We've integrated the standard HelipadHelper into ByoDC. This provides better upload stability and ByoDC now respects the HelipadHelper configuration of the cashbox.
```

File: middleware/2023-03-15-1.3.45.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+If you are experiencing `OutOfMemory` exceptions during helipad upload please configure the HelipadHelper like described [here](TODO-link-to-KBA).
```
Comment (TSchmiedlechner) [inline middleware/2023-03-15-1.3.45.md:20]: I'm not sure if this KBA exists 🥲

File: middleware/2023-03-15-1.3.45.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+## Feature: Update ByoDC base images
+We've update the ByoDC docker base images to debian bullseye and ubuntu jammy.
```
Comment (TSchmiedlechner) [inline middleware/2023-03-15-1.3.45.md:23]: I always have to look up the numbers 😄 
```suggestion
We've update the ByoDC Docker base images to Debian 11 (Bullseye) and Ubuntu 22.04 LTS (Jammy Jellyfish)
```

File: middleware/2023-03-15-1.3.45.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+## Feature: Better ByoDC helm-chart installation process
+
+We've changed how the Emissary Ingress is installed in the ByoDC helm-chart.
+
+This should make installation and troubleshooting more transparent. Also this allows for multiple ByoDC installations in one cluster (See [this guide](https://github.com/fiskaltrust/product-de-bring-your-own-datacenter/blob/master/how-to-multiple-installations.md) for more information).
+
+:::danger
+
+Please follow the [Migration Guide](https://github.com/fiskaltrust/helm-charts/blob/master/bring-your-own-datacenter/MIGRATION.md#v1345) when *updating* to this version of BYODC. 
+Attempting the update without following the steps described there could destroy your BYODC installation.
```
[the remark this answers is not in the document]
Comment (volllly) [reply middleware/2023-03-15-1.3.45.md:49]: I've mostly put this there so people really follow the migration guide ^^ I think all probems are actually recoverable

**Review by Bluette-C-Riviere — APPROVED** [review]

**Pull request merged**